test/invariants/fuzz/BMath.t.sol

```diff
@@ -17,9 +17,10 @@ contract FuzzBMath is EchidnaTest {
    * @dev Reducing BPOW_PRECISION may allow broader range of values increasing the gas cost
    */
   uint256 constant MAX_BALANCE = 1_000_000e18;
-  uint256 constant MIN_BALANCE = 1e18;
-  uint256 constant MIN_AMOUNT = 1e6;
-  uint256 constant MAX_TOLERANCE = 1e12;
+  uint256 constant MIN_BALANCE = 100e18;
+  uint256 constant MIN_AMOUNT = 1e12;
+  uint256 constant TOLERANCE_PRECISION = 1e18;
+  uint256 constant MAX_TOLERANCE = 1e18 + 1e15; //0.1%
```

Comment:
what this whole thing means is 'ensure results are 0.1% from each other in every case where the amount to be traded is at most 8 orders of magnitude away from the pool's balance' which I think is pretty reasonable

used the same MIN_BALANCE as in bconst